packages/tailwindcss/src/compat/config/types.ts

```diff
@@ -18,7 +18,7 @@ export type ThemeConfig = Record<string, ThemeValue> & {
 
 export interface ResolvedConfig {
   theme: Record<string, Record<string, unknown>>
-  plugins: PluginWithConfig[]
```

Comment:
This one feels wrong to change. In theory this type could be exposed to the user (it might be with the `config()` function actually — that works in v4 right?)

Reply:
Ah yep you are right this is currently exposed and this does break user space. Need to make it even uglier and emit this as a separate `Set<>` like you suggested via DM. 😐

Reply:
We could also potentially put it into `PluginWithConfig` since that is already an object with a `handler` function in v3, adding a new property there would not be an issue I think then. WDYT?

Reply:
Yeah lets do that